src/main/java/UI.java

```diff
@@ -18,28 +18,28 @@ public class UI {
     // Patient
     private static final String PATIENT_MAIN_MENU =
             "This is the Patient Main Menu!" + System.lineSeparator()
-            + "List of commands: "
-            + UI.PATIENT_ADD
-            + UI.PATIENT_VIEW_ALL
-            + UI.PATIENT_RETRIEVE
-            + UI.PATIENT_EDIT
-            + UI.RETURN_TO_MAIN
-            + UI.EXIT_PROGRAM;
+                    + "List of commands: "
```

Comment:
May consider removing the extra tabs as 8 spaces should be fine.

Reply:
not my code lol but sure, you can check next time who wrote it...

Reply:
but you can look at the file changed, you can see the extra tabs you commited